v2/requirements.md

```diff
@@ -79,16 +80,15 @@ All other terms in **bold** will be defined in one or more ToIP glossaries in th
     - iii. Operational **requirements**.
     - iv. Legal contracts.
 7. MUST specify in its **EGF** (or in any referenced **credential governance framework**) **requirements** for:
-    - i. An **authorized issuer**, including:
-        - a. The **EGF URI** that MUST be included as a **claim** in any authorized **credential**.
-        - b. The **credential type URI** that MUST be used for any authorized **credential**.
-    - ii. An **authorized verifier**, including:
-        a. The **presentation type URI** that an **authorized verifier** MUST use for any authorized **presentation request**.
+    - i. all `authorization` values that are used by the trust registry.
+    - ii. all Assurance Levels, specified with unique names, that are service by the trust registry.
```

Comment:
What is an Assurance Level?

Reply:
@andorsk updating ToIP glosarry with a definition. The PCTF defines it as "In the Pan-Canadian Trust Framework™ (PCTF), a Level of Assurance (LoA) represents the level of confidence an Entity may place in the processes and other conformance criteria defined in any given component of the PCTF. Levels of Assurance are elemental in creating networks of trust."

@a-fox assurance levels would be defined in the EGF and be ecosystem-specific.

Reply:
I think the "Assurance Level" concept can be offered as something which can be adopted as is or an equivalent in the jurisdiction is adopted. Reminding the reader that LoA is a critical part of this is necessary.